src/middlewares/authorization.js

```diff
@@ -24,7 +22,7 @@ const handleJWT = (req, res, next, roles) => async (err, user, info) => {
   }
 
   // see if user is authorized to do the action
-  if (!roles.includes(user.role)) {
+  if (role && role.includes('admin') && !user.admin) {
```

Comment:
This method was used to add new roles to user model. Not just admin. There maybe an editor, moderator etc as many as possible.

In route definition we can add which roles can see the resouece. Like say you want a protected route only for admins and editors.

we can simply put

`route.post('/page-edit', authorize(['admin', 'editor']))`